backend/user/views.py

```diff
@@ -109,14 +119,32 @@ def post(self, request):
                 notificationsetting__enabled=True,
             )
             .exclude(token="")
+            .values_list("user__username", "token", "dev")
         )
 
+        # unpack list of tuples
```

Comment:
I think these few lines and the query on L114 are also duplicated in `send_shadow_notifs.py`. Might be a good idea to de-duplicate them by putting it in `notifications.py`